examples/js/loaders/GLTFLoader.js

```diff
@@ -277,9 +277,9 @@ THREE.GLTFLoader = ( function () {
 		var extension = ( json.extensions && json.extensions[ EXTENSIONS.KHR_LIGHTS_PUNCTUAL ] ) || {};
 		var lights = extension.lights || {};
```

Comment:
var lights = extension.lights || [];

not `{}`, but `[]` here?

Reply:
Oops yes, thanks! Fixed this and did some other cleanup.